app/models/megaphone/episode.rb

```diff
@@ -7,6 +7,9 @@ class Episode < Integrations::Base::Episode
     SOURCE_ATTRIBUTES = %i[source_media_version_id source_size source_fetch_count source_url source_filename]
     attr_accessor(*SOURCE_ATTRIBUTES)
 
+    # Default duration to publish early on megaphone
+    PUBLISH_EARLY_DURATION = 30.minutes.freeze
```

Comment:
kind of like our feed offset, but publish in megaphone a bit early

Reply:
Is there a reason not to use the db field?  Just to prevent other unintended consequences, or ensure it's not overridable?

Reply:
If I set and use the db field, it has the unintended consequence of applying to when episodes are published in our feed as well, which isn't the point here - it might be safe? Since our megaphone feed is not relying on the offset for DTR access, it might be ok? When I looked through the places where `episode_offset_seconds` was used, it seemed safer to avoid it, but it looks ok when I look again now...